src/skills-builder/skills-builder-modal/select-preferences/test/SelectPreferences.test.jsx

```diff
@@ -2,8 +2,13 @@ import {
   screen, render, cleanup, fireEvent,
 } from '@testing-library/react';
 import { mergeConfig } from '@edx/frontend-platform';
+import { sendTrackEvent } from '@edx/frontend-platform/analytics';
 import { SkillsBuilderWrapperWithContext, dispatchMock, contextValue } from '../../../test/setupSkillsBuilder';
 
+jest.mock('@edx/frontend-platform/analytics', () => ({
+  sendTrackEvent: jest.fn(),
```

Comment:
Is this relevant if you've got the mock in `setupSkillsBuilder.jsx`?

Reply:
Good catch! It will work if it's mocked individually in each file. Having it in `setupSkillsBuilder` is actually not doing anything for us now so that can be removed at the very least. I'm not sure how to get the tests to mock using just the setup file. It doesn't seem to pick up on it being a `jest` function

Reply:
Setup is great, when it works the way we want it to. 😆